src/DotNetWorker.Core/Context/Features/DefaultFunctionInputBindingFeature.cs

```diff
@@ -53,6 +53,13 @@ public async ValueTask<FunctionInputBindingResult> BindFunctionInputAsync(Functi
                 {
                     FunctionParameter parameter = context.FunctionDefinition.Parameters[i];
 
+                    // Option 1: If the parameter does not have a default value and is nullable type, set it to null.
+                    if (!parameter.HasDefaultValue && parameter.IsReferenceOrNullableType)
```

Comment:
Actually on second thought, not having a default value does not necessarily mean that its an empty/null item. So maybe we could check if source is null or empty here then check for these parameters? But that might not be any difference to just checking for string empty in our converters